README.md

```diff
@@ -15,22 +15,42 @@ With the release of AndroidX, Android Support is now considered deprecated and w
 
 ## Binding Policies
 
-- This repository binds over 100 AndroidX libraries that are published to [NuGet.org](https://nuget.org). The full package list can be found in [config.json](config.json).
+- This repository binds over 200 (2022-09) AndroidX libraries that are published to [NuGet.org](https://nuget.org). The full 
+  package list can be found in [config.json](config.json).
 - AndroidX Java artifacts come from [Google's Maven Respository](https://maven.google.com/web/index.html#).
+- dependency Maven artifacts come from different maven repositories mostly:
+  - [Google's Maven Respository](https://maven.google.com/web/index.html#) and
+  - [Maven/Sonatype Central](https://repo1.maven.org/maven2/)
 - Google's release notes for AndroidX libraries are available [here](https://developer.android.com/jetpack/androidx/versions/stable-channel).
-- The major/minor/patch version numbers mirror the AndroidX library version. For example, the NuGet `Xamarin.AndroidX.Core 1.3.2.1` binds version `1.3.2` of the AndroidX library `androidx.core:core`.
+- The major/minor/patch version numbers mirror the AndroidX library version. For example, the NuGet `Xamarin.AndroidX.Core 1.3.2.1` 
+  binds version `1.3.2` of the AndroidX library `androidx.core:core`.
   - The revision version number is used when a new NuGet needs to be built but the AndroidX library has not been updated.
-- We endeavor to release updated NuGets within a few weeks after new AndroidX releases, however large changes occasionally require more time.
+- We endeavor to release updated NuGets within a few weeks after new AndroidX releases, however large changes occasionally require 
+  more time.
 - In general, we do not bind pre-release libraries. As their API is not stable yet, it results in too much rework.
 
+### Details
```

Comment:
I would highly recommend we not introduce these lists.  Our users are programmers, they should be able to read `json` and can find this information in `config.json` as stated above.

Introducing these means additional files that must be updated with every change we make.

Reply:
> Introducing these means additional files that must be updated with every change we make.

Those tables are automatically generated. Nothing manual, but readme.md that's the reason I added them as separate markdown files that contain just `config,json` in tabular form.

https://github.com/xamarin/AndroidX/blob/docs-updates/utilities.cake#L329-L398

Reply:
and yes I run `utilities.cake` locally after `ci`, so the files will be updated regularly. 

Actually this is side product of my work on dependency graphs for both maven and nuget and other utilities.